frontend/src/routes/routesConfig.tsx

```diff
@@ -0,0 +1,42 @@
+import SignIn from "../pages/auth/signin/SignIn";
+import SignUp from "../pages/auth/signup/SignUp";
+import React, {ReactNode} from "react";
+import Profile from "../pages/main/profile/Profile";
+import TopUp from "../pages/main/topup/TopUp";
+import Transactions from "../pages/main/transactions/Transactions";
+import Main from "../pages/main/Main";
+
+export interface IRoute {
+    path: string
+    element: ReactNode
+}
+
+export const mainRoutes: IRoute[] = [
+    {
+        path: "/profile",
```

Comment:
I suggested to use constants. Why did you skip it? It can help you to improve readability by using these constants instead of getting path from array by index